Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3.9
+
+ARG DEBIAN_FRONTEND="noninteractive"
+
+RUN apt-get update -qq && \
+    apt-get install -y -qq --no-install-recommends \
+        git && \
+    rm -rf /var/lib/apt/lists/*
+
+ARG TEMPLATEFLOW_HOME="/templateflow"
+
+RUN pip3 install nilearn==0.9.2 templateflow pybids h5py tqdm&& \
+    mkdir -p /code && mkdir -p /templateflow
+
+WORKDIR /code
+
+RUN python3 -c "from templateflow.api import get; get(['MNI152NLin2009cAsym', 'MNI152NLin6Asym'])"
+
+COPY [".", "/code"]
+
+RUN pip install --upgrade pip && pip3 install -e .
+
+ENV TEMPLATEFLOW_HOME=${TEMPLATEFLOW_HOME}
+
+RUN git submodule update --init --recursive && python3 tools/download_templates.py
```

Comment:
This and L17 will run into issues on beluga: the computing node doesn't have access to the network. Plus this will not be needed every time, probably only needed when people run the tool for the first time. It might be useful to have an extra flag for the bids app for people to fetch the templates only. WDYT?